frontend/js/src/artist-similarity/src/Input.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+interface InputProps {
+    fetchData: Function;
+    setLimit: Function;
+}
+
+const Input = (props: InputProps) => {
+
+    const handleInput = (event: React.FormEvent<HTMLFormElement>): void => {
+        event.preventDefault();
+        const form = event.target as HTMLFormElement;
```

Comment:
I see another `as` band-aid :)
Could this possibly be undefined? then we should make sure we plan for that rather than shut Typescript up :)

Reply:
Fixed, using event.currentTarget gets rid of the as